ledger/participant-integration-api/src/main/scala/platform/store/cache/StateCache.scala

```diff
@@ -130,6 +163,14 @@ private[platform] case class StateCache[K, V](cache: Cache[K, V], registerUpdate
           logger.error(s"Expected pending updates tracker for key $key is missing")
         }
     )
+
+  private def truncateValueForLogging(value: V) = {
```

Comment:
No need to change, but I could see this guy as some kind of a logging library...or makes me wonder whether this could be achieve already by some logging feature with {} :thinking: